### ISSUE TITLE
Card name displaying for disconnecting spectator

### ISSUE THREAD

**Issue by FarCryFromHuman** [mtgred/netrunner#1725]
This is a weird one... Note the surrounded text below.
![image](https://cloud.githubusercontent.com/assets/12355388/16592557/820f721a-429e-11e6-808e-fd8a4a5ba970.png)

I don't know what the user's actual name was, but upon disconnection his name was printed as Singularity, complete with orange text and a correctly working on-hover card pop-up.

**Comment by nealterrell** [comment]
User names that match card names do display as card links, so it's most likely his name is literally Singularity.

**Comment by FarCryFromHuman** [comment]
Yep that would do it. I'm going to label this as a cosmetic issue.

**Comment by erbridge** [comment]
I actually think this is a feature.

**Comment by zaroth** [comment]
Kind of related to #343, if we take care of system messages being classified as system messages, might as well make them look different.
